examples/example.c

```diff
@@ -290,6 +290,12 @@ main(int argc, char **argv)
         sentry_options_set_crashpad_wait_for_upload(options, true);
     }
 
+    if (has_arg(argc, argv, "attach-view-hierarchy")) {
+        // assuming the example / test is run directly from the cmake build
+        // directory
+        sentry_options_add_view_hierarchy(options, "./CMakeCache.txt");
+    }
```

Comment:
Since this is now an option specifically for that use case, I think it is fair to provide an actual `view-hierarchy.json` as a test fixture. See how you can bind the update of that file in the build directory to the `sentry_example` target: 

https://github.com/getsentry/sentry-native/blob/dc4d53dd9a5c225063bd5edfcb8387b2bb2e4912/CMakeLists.txt#L659-L660

You can use a JSON as minimal as in the relay test. There is no need to parse in our integration test (although I am certainly not against it either).